app/components/icons-generated/Undo.js

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react'
+const SvgUndo = (props) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    height={24}
+    viewBox="0 -960 960 960"
+    width={24}
+    {...props}
+  >
+    <path
+      className="tag_svg__gray"
```

Comment:
good try 👍 but this needs to be set in the SVG file itself as `class="gray"`, not a manual change in generated file

I pushed a fix commit to your forked repo, please pull before making more changes, thanks!